### DISCUSSION TITLE
Using Glide on specific images in image list/arrays

### DISCUSSION THREAD

**Discussion by Sennik** [statamic/cms#4361] [Troubleshooting] [answered]
I have a custom gallery layout where 6 images together form a styled gallery with all images have specific sizes. Images uploaded by users could have different ratio's so I have to crop them before using them in this gallery. 

![2021-10-02 11_00_50-Home](https://user-images.githubusercontent.com/5636394/135710056-2b0a8705-9570-4406-9904-ca71895cef99.png)

Glide to the rescue (I thought)

**Why is this:** 
```
<img src="{{ gallery_image:0 }}">
<img src="{{ glide:gallery_image width="250" }}">
```
**working but:**  
```
<img src="{{ glide:gallery_image:0 width="250" }}">
```
not?

It should by logical to use glide on both IMHO. How can I achieve this?

**Comment by jackmcdade** [comment, marked as answer]
Try this:

```
<img src="{{ glide :src="gallery_image:0" width="250" }}">
```

**Reply by Sennik** [reply]
Works! Thank you!
